specification/resourcemover/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/resourcemovercollection.json

```diff
@@ -1304,6 +1309,15 @@
         },
         "provisioningState": {
           "$ref": "#/definitions/ProvisioningState"
+        },
+        "errors": {
+          "description": "Defines the move collection errors.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/MoveResourceError"
```

Comment:
Should we instead change this object to MoveError?

Reply:
yes, we can do that thought of taking that as a different PR, wanted to unblock the correctness.

Reply:
@innosam It would be better to change it in this PR. Otherwise, it would be considered as breaking change since you change a response object, which could block merge in the next time.

Reply:
Basically he is suggesting to rename the structure, not change the reference to another object. That's not a priority change for us, we can see if possible take in in next pr or in the next version.
Let me know if there are any further clarifications required here.